### DISCUSSION TITLE
8月1日起，LeanCloud 国际版共享域名不再向中国大陆提供服务

### DISCUSSION THREAD

**Discussion by imjohncoker** [walinejs/waline#1203] [Q&A] [answered]
8 月 1 日起，LeanCloud 国际版共享域名不再向中国大陆提供服务
为履行合规责任，降低平台风险，LeanCloud 国际版共享域名将于 2022 年 8 月 1 日起不再向中国大陆的最终用户提供服务，国际版共享域名仅服务于海外用户。

影响范围
8 月 1 日以后，国内的 IP 无法通过共享域名调用国际版的数据存储、即时通信、云引擎等全部服务，请求会响应错误码 1020。

开发者需要做什么
如果国际版应用主要服务海外用户，那么不需要做任何处理。如果应用的用户主体是在中国大陆，需要使用自己的域名或者将应用数据迁移至国内节点，下面依次介绍这两种方式。

1. 使用自己的域名

参考[域名绑定指南](https://leancloud.cn/docs/custom-api-domain-guide.html)文档。使用数据存储、即时通信、推送与短信服务的用户，需要在控制台绑定 API 自定义域名；使用云引擎服务的用户，需要在控制台绑定云引擎自定义域名。

需要注意，如果你在 2020 年 10 月之前绑定过自定义域名，需要修改域名解析至控制台上显示的最新的 CNAME 地址，否则从中国大陆的访问会被禁止！新的地址可以在 应用设置 > 域名绑定 处看到。

2.将应用数据迁移至国内节点

可以将应用数据迁移至国内节点（华北节点或华东节点），数据的迁移方式是将[国际版数据导出后再导入到国内节点](https://leancloud.cn/docs/account-and-dashboard-faq.html#hash-1060823898)的应用下。

需要注意：

数据存储服务的结构化数据支持导出与导入。

即时通信服务的会话记录与聊天记录无法导出。

云引擎需要重新部署。

短信服务的签名与模板需要重新审核。

推送厂商设置、开发证书上传、敏感词库上传等应用的设置信息需要重新配置。

国内节点的应用需要实名认证后使用，并且需要 [绑定自己的已备案域名](https://leancloud.cn/docs/custom-api-domain-guide.html)。

另外，工信部规定，网站接入多个云服务商时，需要在各云服务商处接入备案。 如果应用使用的自定义域名没有办理接入备案，请通过[工单](https://leanticket.cn/)提交相应资料，以便我们协助办理接入备案。提交工单时，分类请选择 域名与备案 > 备案接入（企业） 或 备案接入（个人）。详见[接入备案常见问题](https://forum.leancloud.cn/t/topic/24998)。

如有疑问，欢迎通过[工单](https://www.leanticket.cn/)或[论坛](https://forum.leancloud.cn/latest)和我们联系。

The LeanCloud Team

本邮件由 LeanCloud 系统自动发出，请勿直接回复
如何处理？

**Comment by lizheming** [comment, marked as answer]
LeanCloud 国际版 IP 限制的这个事情，对 Waline 没有影像哈，Waline 是用 Vercel 访问的 LeanCloud，Vercel 是国外 IP 所以不受影响。

**Reply by Superoutman** [reply]
我的评论不能显示了，开了全局代理就能显示，是不是说明受到影响了？

**Reply by lizheming** [reply]
> 我的评论不能显示了，开了全局代理就能显示，是不是说明受到影响了？
https://github.com/walinejs/waline/issues/1334

**Reply by conge** [reply]
> 我的评论不能显示了，开了全局代理就能显示，是不是说明受到影响了？

如果你是用vercel部署，且没有用自定义域名，那很可能是因为是受到 vercel.app 域名被墙的影响。参考：https://blog.lingerbhw.cn/posts/dbf4d938/

**Reply by Superoutman** [reply]
感谢两位 @lizheming @conge
